modules/Bio/EnsEMBL/Compara/PipeConfig/CreateSpeciesTree_conf.pm

```diff
@@ -67,7 +72,7 @@ sub default_options {
         'master_db'          => 'compara_master',
 
         'representative_species' => undef,
-        'taxonomic_ranks' => ['order', 'class', 'phylum', 'kingdom'],
+        'taxonomic_ranks' => ['genus', 'family', 'order', 'class', 'phylum', 'kingdom'],
```

Comment:
Does it work for Vertebrates with both _genus_ and _family_ in the list ?

Reply:
Yes, a filter is put in later to ensure that for the large species_sets the taxonomic ranks that contain too few species are eliminated. Tests have been run on both the rice species set and the default vertebrates. The default vertebrates tree from these tests and the release 99 tree of default vertebrates have been compared and shown to be identical.